### ISSUE TITLE
feature: multiple profiles

### ISSUE THREAD

**Issue by aradzie** [aradzie/keybr.com#10]
This idea came from a user:

> Awesome product, thanks for what you've built!
>
> I want to practice in different environments: for example, with an ortholinear keyboard on my desktop and maintain speed on my laptop. Or, in the future, switching between qwerty and colemak.
>
> But, these require different learning profiles, where we don't compare the results of one context to the other.
>
> Is there a way to do this?

**Comment by sehrgut** [comment]
This is exactly the feature request I came here to post. I use tags on Monkeytype to separate my different keyboards and layouts, and something like that here would be super helpful.

**Comment by sdeck51** [comment]
I wanted to bump this as I've been using the site for learning colmakdh on a glove80, but recently bought a macbook which is staggered and am using ansi instead of matrix view\layout. It would be great to be able to practice both on the same account without having to change the settings every time I switch.

**Comment by Blake-LeBlanc** [comment]
Yes, this would be great! I'd like to use keybr to learn an alternate layout like colmak-dh but I don't want it polluting my progress/results with qwerty.
